### PR TITLE
Iteration 1 henley

### PR DESCRIPTION
**Pull request by HenleyCH**
Initial pull request for iteration-1-henley feature branch

### CODE REVIEW

File: src/FixBookControl.java [hunk 1]
```diff
@@ -19,7 +19,7 @@ public void Set_Ui(FixBookUI ui) {
 			throw new RuntimeException("FixBookControl: cannot call setUI except in INITIALISED state");
 		}	
 		this.UI = ui;
-		ui.Set_State(FixBookUI.UI_STATE.READY);
+		ui.Set_State(FixBookUI.UIState.READY);
```
Comment (cmougios-ITC205) [inline src/FixBookControl.java]: Can we clarify whether we are using camelBack style - Should it be UIState or UiState?

File: src/FixBookUI.java [hunk 2]
```diff
@@ -47,7 +47,7 @@ public void run() {
                 break;
 
             case FIXING:
-                String AnS = input("Fix Book? (Y/N) : ");
+                String AnS = askQuestion("Fix Book? (Y/N) : ");
                 boolean FiX = false;
                 if (AnS.toUpperCase().equals("Y")) {
```
Comment (cmougios-ITC205) [inline src/FixBookUI.java]: should askQuestion cover both upper and lowercases? if the user enters lower case y/n, does it bomb out? If so, perhaps need to put in place that if lowercase, then change case to upper?

**Comment by cmougios-ITC205** [comment]
FixBookControl - 
Line 3 - var UI - need to confirm if Ui should be used instead

Block style code- need to confirm whether we are using next-line or end-of-line braces - You are currently using end-line-braces, I'm using next-line.

line 17 - method name should it be setUi or setUI & fixBookUI - Are we conforming to the camelBack coding guidelines?


FixBookUI - 
line 6 - enum UiState or UIState?

File: src/FixBookUI.java (hunk 2 above)
Comment (HenleyCH) [reply src/FixBookUI.java]: .toUpperCase() is there to do that.

File: src/FixBookControl.java (hunk 1 above)
Comment (HenleyCH) [reply src/FixBookControl.java]: 'UI' is always capitalised. This is a rule we verbally decided on in-meeting and again discussed on the Discord message thread.
I am currently creating an updated code style guidelines document on Google Drive, to ensure the rules are all in one place.

**Review by cmougios-ITC205 — APPROVED** [review]
Please proceed to next step

**Pull request merged**